core/src/main/java/org/mskcc/cbio/oncokb/util/ValidationUtils.java

```diff
@@ -134,6 +134,42 @@ public static JSONArray checkEvidenceDescriptionReferenceFormat() {
         return data;
     }
 
+    public static JSONArray checkEvidenceDescriptionHasOutdatedInfo() {
+        // the goal is to find whether the description includes outdated info
+        //
+        // Check one: drugs that in the production but no longer included in the latest data
+        JSONArray data = new JSONArray();
+
+        DrugBo drugBo = ApplicationContextSingleton.getDrugBo();
+        Set<Drug> drugsWithEviAssoc = new HashSet<>();
+        CacheUtils.getAllEvidences().forEach(evidence -> evidence.getTreatments().forEach(treatment -> drugsWithEviAssoc.addAll(treatment.getDrugs())));
+
+        List<Drug> allDrugs = drugBo.findAll();
+        Set<Drug> drugsWithoutEviAssoc = allDrugs.stream().filter(drug -> !drugsWithEviAssoc.contains(drug)).collect(Collectors.toSet());
+
+        Set<String> drugNameWithSynonyms = new HashSet<>();
+        for (Drug drug : drugsWithoutEviAssoc) {
+            drugNameWithSynonyms.addAll(drug.getSynonyms());
+            drugNameWithSynonyms.add(drug.getDrugName());
+        }
+        Set<EvidenceType> evidenceTypesToCheck = new HashSet<>();
+        evidenceTypesToCheck.add(EvidenceType.MUTATION_EFFECT);
```

Comment:
If I remember correctly, it is fine to mention a deleted drug in the mutation effect summary. 
Do you think we should still include this or its valuable to let the curators know about these summaries?

Reply:
@calvinlu3 Good point, ME is tricky. Let's set it up for now. We will do one data release, then decide whether to exclude.